apps/judicial-system/web/src/shared-components/AccordionItems/CourtRecordAccordionItem/CourtRecordAccordionItem.tsx

```diff
@@ -63,12 +63,14 @@ const CourtRecordAccordionItem: React.FC<Props> = ({ workingCase }: Props) => {
       <AccordionListItem title="Krafa" breakSpaces>
         <Text>{workingCase.prosecutorDemands}</Text>
       </AccordionListItem>
-      <AccordionListItem
-        title={formatMessage(m.sections.courtAttendees.title)}
-        breakSpaces
-      >
-        <Text>{workingCase.courtAttendees}</Text>
-      </AccordionListItem>
+      {workingCase.courtAttendees && (
```

Comment:
Consider trimming courtAttendees. We had to do that when generating the pdf server side.